requirements.txt

```diff
@@ -16,6 +16,7 @@ pyramid==1.7.3
 pyramid-multiauth==0.9.0
 pyramid-tm==1.1.1
 python-dateutil==2.6.0
+PyYAML==3.12
```

Comment:
And also because we had this discussion for kinto-wizard and we made this choice there: https://github.com/Kinto/kinto-wizard/blob/master/setup.py